packages/flutter_tools/test/create_test.dart

```diff
@@ -36,6 +37,37 @@ void main() {
       return _createAndAnalyzeProject(temp, <String>['--with-driver-test']);
     });
 
+    // Verify content and formatting
+    testUsingContext('content', () async {
+      Cache.flutterRoot = '../..';
+
+      CreateCommand command = new CreateCommand();
+      CommandRunner runner = createTestCommandRunner(command);
+
+      int code = await runner.run(<String>['create', '--no-pub', temp.path]);
+      expect(code, 0);
+
+      void expectExists(String relPath) {
+        expect(FileSystemEntity.isFileSync('${temp.path}/$relPath'), true);
+      }
+      expectExists('lib/main.dart');
+      await for (FileSystemEntity file in temp.list(recursive: true)) {
```

Comment:
`listSync()` here instead? You can avoid the `await` on the for statement, and will make this slightly simpler at runtime.